pkg/activator/throttler.go

```diff
@@ -148,7 +148,7 @@ func (t *Throttler) Try(rev RevisionID, function func()) error {
 
 func (t *Throttler) activatorEndpointsUpdated(newObj interface{}) {
 	endpoints := newObj.(*corev1.Endpoints)
-	t.numActivators = resources.ReadyAddressCount(endpoints)
+	t.numActivators, _ = resources.NewFixedEndpointsCounter(endpoints).ReadyCount()
```

Comment:
It feels to me like we really should keep the `resources.ReadyAddressCount` function intact instead of replacing it everywhere. Ignoring the error is fine here because you know it will never error, but that really should be encoded in the function's signature, in that it doesn't return an error at all.

Reply:
I think on reflection that you're right. I had this because I wanted to have a consistent use of these two different kinds of count, but ignoring magical never-never errors seems like a worse outcome. I'll walk it back to `ReadyAddressCount`.